mscore/cloud/uploadscoredialog.cpp

```diff
@@ -10,12 +10,14 @@
 //  the file LICENCE.GPL
 //=============================================================================
 
+#include <QMessageBox>
```

Comment:
not in all.h?

Reply:
QMessageBox is in `all.h, it added it here, so that the IDE (QtCreator) would work correctly

Reply:
I see. We may want that in many more places...